psalm.xml

```diff
@@ -0,0 +1,185 @@
+<?xml version="1.0"?>
+<psalm
+    totallyTyped="false"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="vendor/vimeo/psalm/src/config.xsd"
```

Comment:
Will this work if one is using `psalm.phar` instead of having it as a composer dependency?

Reply:
the schemaLocation is just so IDEs can do autocomplete on the XML.

Reply:
Okay.